packages/frint-props/src/withStore.js

```diff
@@ -0,0 +1,79 @@
+import { from } from 'rxjs/observable/from';
+import { of } from 'rxjs/observable/of';
+import { merge } from 'rxjs/observable/merge';
+import { map } from 'rxjs/operators/map';
+import { switchMap } from 'rxjs/operators/switchMap';
+import { scan } from 'rxjs/operators/scan';
```

Comment:
This would definitely look better with RxJS 6 😃:

```js
import { from, of, merge, map, switchMap, scan } from 'rxjs';
```

Reply:
I badly hope they also support this in RxJS v6:
 
```js
import { map } from 'rxjs/map';
import { scan } from 'rxjs/scan';
```

Otherwise our internal concept of bundling multiple apps with same config would be a total nightmare 😅